src/ResourceManager/Compute/Commands.Compute/VirtualMachine/Config/NewAzureVMConfigCommand.cs

```diff
@@ -133,6 +138,11 @@ public override void ExecuteCmdlet()
                 vm.HardwareProfile.VmSize = this.VMSize;
             }
 
+            if (this.EnableUltraSSD.IsPresent)
+            {
+                vm.AdditionalCapabilities = new AdditionalCapabilities(true);
```

Comment:
@hyonholee 
Don't we need to actually assign the ultrassd capability like we do [here](https://github.com/Azure/azure-powershell/pull/7429/files#diff-9c9de3599461d0d75d69ca1fd1421fa5R178)?

Reply:
@praries880 new AdditionalCapabilities(true) creates the object with UltraSSDEnabled = true.  The links is for Update, where it creates the object and then sets the Boolean value as the same as the passed value.

Reply:
@hyonholee  thanks for clarifying